anta/cli/__init__.py

```diff
@@ -55,11 +55,15 @@
     ),
     callback=setup_logging,
 )
+@click.option("--ignore-status", show_envvar=True, default=False, help="Always exit with success", is_eager=True)
```

Comment:
is_eager is not needed here.
Also, defining options like this means you wants a boolean as parameter.
```
anta --help
  --ignore-status BOOLEAN         Always exit with success  [env var:
                                  ANTA_IGNORE_STATUS]
  --ignore-error BOOLEAN          Only report failures and not errors  [env
                                  var: ANTA_IGNORE_ERROR]
```
You need
```suggestion
@click.option("--ignore-status", is_flag=True, show_envvar=True, default=False, help="Always exit with success", is_eager=True)
```

Could you also update the insecure flag with the is_flag param=True param ?